blueprints/ember-frost-tabs/index.js

```diff
@@ -11,7 +11,8 @@ module.exports = {
   afterInstall: function () {
     return this.addAddonsToProject({
       packages: [
-        {name: 'ember-frost-core', target: '^0.29.0'}
+        {name: 'ember-frost-core', target: '^0.29.0'},
+        {name: 'ember-simple-uuid', target: '0.1.4'}
```

Comment:
I disagree, we've opted to used blueprints instead of dependencies so consumers can choose explicitly what versions of things to use. When you start putting things in the dependencies section of the `package.json` you risk various addons bringing in different versions and an Ember app can have more than one version so one of those will end up winning.